govcd/api_vcd_test.go

```diff
@@ -1093,70 +1093,79 @@ func TestClient_getloginurl(t *testing.T) {
 func TestVCDClient_Authenticate(t *testing.T) {
 	config, err := GetConfigStruct()
 	if err != nil {
-		t.Fatalf("err: %v", err)
+		t.Fatalf("err: %s", err)
 	}
 	client, err := GetTestVCDFromYaml(config)
 	if err != nil {
-		t.Fatalf("err: %v", err)
+		t.Fatalf("err: %s", err)
 	}
 	err = client.Authenticate(config.Provider.User, config.Provider.Password, config.Provider.SysOrg)
 	if err != nil {
-		t.Fatalf("Error authenticating: %v", err)
+		t.Fatalf("Error authenticating: %s", err)
 	}
 }
 
-func (vcd *TestVCD) createTestVapp(name string) (VApp, error) {
+func (vcd *TestVCD) createTestVapp(name string) (*VApp, error) {
+	// ========================= issue#252 ==================================
+	// To be enabled when issue#252 is resolved.
```

Comment:
Maybe worth setting `TODO` so that automated tools pick it up. Or maybe it is cheaper to put a note in the issue itself.

Reply:
Done